### PR TITLE
enh(api): add Postman tutorial to get a hands-on API v2

### PR DESCRIPTION
**Pull request by cgagnaire**

### CODE REVIEW

File: fr/api/rest-api-v2.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+Centreon Web Rest API v2 a pour objectif de remplacer la première version.
+
+Pour commencer à l'utiliser, référez-vous à la [page
+dédiée](https://docs.centreon.com/api/centreon-web/).
+
+## Créer une collection Postman depuis la définition OpenAPI
+
+Afin de manipuler l'API plus efficacement ou seulement pour comprendre comment
+elle fonctionne, il n'y a rien de plus pratique qu'une collection Postman.
```
Comment (Moujimouja) [inline fr/api/rest-api-v2.md:16]: Could be nice to add the link to the Postman website

File: fr/api/rest-api-v2.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+identifiants.
+
+Depuis l'espace de travail, cliquer sur l'icone **Manage Environments**.
+
+![image](../assets/api/postman-environment.png)
+
+Cliquer sur **Add** et definir les variables comme suit avec les informations
+de la plateforme :
+
+![image](../assets/api/postman-environment-add.png)
```
Comment (Moujimouja) [inline fr/api/rest-api-v2.md:59]: same comment > why {{version}} is not a variable to be defined ?

**Review by Moujimouja — COMMENTED** [review]
Quick comment to understand why {{verison}} doesn't need to be defined they it's 👍

File: en/api/rest-api-v2.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+To be able to communicate with your platform, the collection will use
+environment variables to define access information and credentials.
+
+From your workspace, click on the **Manage Environments** icon.
+
+![image](../assets/api/postman-environment.png)
+
+Click on **Add** and set the variables as below with your platform information:
+
+![image](../assets/api/postman-environment-add.png)
```
[the remark this answers is not in the document]
Comment (cgagnaire) [reply en/api/rest-api-v2.md:56]: I just added a section about it. It's more of a collection-wide variable.

File: fr/api/rest-api-v2.md (hunk 2 above)
Comment (cgagnaire) [reply fr/api/rest-api-v2.md:60]: I just added a section about it. It's more of a collection-wide variable.

**Review by Moujimouja — APPROVED** [review]

**Pull request merged**